app/objects/quote.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Quote
+  attr_reader :content,
+              :author,
+              :tags,
+              :id
+
+  def initialize(data)
+    @id = data[:_id]
+    @content = data[:content]
+    @author = data[:author]
+    @tags = data[:tags]
+  end
+end
```

Comment:
👍 I like that you're storing the id.. this can definitely come in handy for writing robust tests.